src/defaultsettings.cpp

```diff
@@ -282,6 +282,8 @@ void set_default_settings()
 	settings->setDefault("repeat_joystick_button_time", "0.17");
 	settings->setDefault("joystick_frustum_sensitivity", "170");
 	settings->setDefault("joystick_deadzone", "2048");
+	settings->setDefault("clickable_chat_weblinks", "false");
+	settings->setDefault("chat_weblink_color", "#8888FF");
```

Comment:
This color looks purple, I think `#88c1ff` is better

Reply:
In my experience, this color is less likely to match what a player chooses for a colour_chat csm setting. Besides, I like it.  :]

Reply:
CSM aren't officially supported, and users expect links to be blue